be/src/common/config.h

```diff
@@ -270,6 +270,10 @@ CONF_Bool(enable_low_cardinality_optimize, "true");
 CONF_mBool(enable_compaction_checksum, "false");
 // whether disable automatic compaction task
 CONF_mBool(disable_auto_compaction, "false");
+// whether disable compact when there is a hole in tablet versions
+CONF_mBool(disable_compact_tablet_version_discontinuous, "true");
+// when version incontinuous, if tablet versions exceed `this ratio * max_tablet_version_num`, allow tablet compaction
+CONF_mDouble(allow_compact_tablet_discontinuous_max_rowset_num_ratio, "0.7");
```

Comment:
may be smaller? 0.4 and 200 versions is big enough

Reply:
ok, changed